src/main/java/com/exasol/drivers/ExasolDriverManager.java

```diff
@@ -0,0 +1,98 @@
+package com.exasol.drivers;
+
+import java.util.*;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.exasol.bucketfs.*;
+
+/**
+ * Manager for drivers for external data repositories that are installed on the Exasol Database.
+ */
+public class ExasolDriverManager {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExasolDriverManager.class);
+    private static final String MANIFEST_FILENAME = "settings.cfg";
+    static final String DEFAULT_JDBC_DRIVER_PATH_IN_BUCKET = "drivers/jdbc";
+    static final String MANIFEST_PATH_IN_BUCKET = DEFAULT_JDBC_DRIVER_PATH_IN_BUCKET + BucketConstants.PATH_SEPARATOR
+            + MANIFEST_FILENAME;
+    private final Map<String, DatabaseDriver> installedDrivers = new HashMap<>();
+    private final Bucket bucket;
+
+    /**
+     * Create a new instance of an {@link ExasolDriverManager}.
+     *
+     * @param bucket BucketFS bucket in which the drivers should be installed
+     */
+    public ExasolDriverManager(final Bucket bucket) {
```

Comment:
So we should be able to extract the `ExasolDriverManager` with the bucketfs code one day. :+1:

Reply:
That was part of the plan. Although it is a separate concern that just happens to have a Bucket dependency.